lib/hanami/cli/files.rb

```diff
@@ -42,6 +42,12 @@ def chdir(path, &blk)
         super
       end
 
+      # @since x.x.x
+      # @api private
+      def recommend(message)
+        out.puts("  Recommendation: #{message}")
+      end
```

Comment:
This as part of our `fs` object feels weird. Is there any reason we can't output this from inside our generator class? We could include `out:` as a dependency to that class.

Then it would just be:

```
fs.write # ...
out.puts("Add a namespace...")
```

Reply:
Yup, that makes sense! I did this